src/analytics/redux/tracked_events.js

```diff
@@ -94,14 +142,29 @@ export const trackedEvents = {
 				( { name } ) => ( { block_name: name } )
 			);
 		},
-		moveBlocksUp() {
-			trackBlockMoved( 'move_arrows' );
-		},
-		moveBlocksDown() {
-			trackBlockMoved( 'move_arrows' );
+		startDraggingBlocks() {
+			setIsDraggingBlock( true );
 		},
 		stopDraggingBlocks() {
-			trackBlockMoved( 'drag_and_drop' );
+			setIsDraggingBlock( false );
+		},
+		moveBlocksUp( clientIds ) {
+			trackBlockMoved( clientIds, 'move_arrows_up' );
+		},
+		moveBlocksDown( clientIds ) {
+			trackBlockMoved( clientIds, 'move_arrows_down' );
+		},
+		moveBlocksToPosition(
+			clientIds,
+			_fromRootClientId,
+			_toRootClientId,
+			toIndex
+		) {
+			if ( isDraggingBlock ) {
+				trackBlockMoved( clientIds, 'drag_and_drop' );
+			} else {
+				handleBlockMovedByPosition( clientIds, toIndex );
+			}
```

Comment:
Alternatively, we could check if any block is being dragged via [the `isDraggingBlocks` selector](https://github.com/WordPress/gutenberg/blob/681cde287a2c4907fc74e0347ccfb4b55d0f8ff1/packages/block-editor/src/store/selectors.js#L1284-L1293). This way we wouldn't need to keep the `isDraggingBlock` value and update it by listening to `startDraggingBlocks` and `stopDraggingBlocks`.

Reply:
Updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4833/commits/2037f66df75d33a68292e074e30fe4d1b053e7f6